components/camel-google-pubsub/src/main/java/org/apache/camel/component/google/pubsub/consumer/CamelMessageReceiver.java

```diff
@@ -0,0 +1,62 @@
+package org.apache.camel.component.google.pubsub.consumer;
```

Comment:
Can you please run your build using the source check `mvn -Psourcecheck clean install` to fix the checkstyle issues, for example the missing license header in this class